rs/sns/swap/canister/canister.rs

```diff
@@ -390,6 +362,7 @@ fn create_real_icp_ledger(id: CanisterId) -> ic_nervous_system_common::ledger::I
 /// Returns a real ledger stub that communicates with the specified
 /// canister, which is assumed to be a canister that implements the
 /// ICRC1 interface.
+#[allow(unused)]
```

Comment:
Y tho?

In general, I think these should be accompanied by a comment that explains why the thing that you usually want does not work in this case (and is therefore disabled).

Reply:
True, I don't have anything against removing it actually